flybywire-aircraft-a320-neo/effects/LIGHT_A32NX_WingLarge.fx

```diff
@@ -65,9 +65,9 @@ MinProjSize=0.2
 
 [LightAttributes.0]
 Type=spot
-Size=0.1
-Range=20
-Intensity=50
+Size=0.8
+Range=100
+Intensity=100
```

Comment:
Why is the "physical" size of the "bulb" increased x 8, the range by x 5 and the intensity with x 2 (actually x 4 in total when combined/calculated with the volumetric light).

Perhaps the light for the leading edge should be better focused and repositioned. (These light does not need to originate from the modeled light on fuselage, feel free to reposition/add more)